backend/src/db/db.client.ts

```diff
@@ -0,0 +1,32 @@
+require("dotenv").config();
+const { MongoClient } = require("mongodb");
+import logger from '../util/winston.createLogger'
+
+
+const { dbURI } =  process.env as {
```

Comment:
jak wyciągasz jednąrzecz to zrobiłbym to tak const dbURI: string = process.env.dbURI

Reply:
const { dbUri }: { [key: string]: string  } = process.env